Security/Firewall/OAuth2Listener.php

```diff
@@ -53,12 +52,12 @@ public function __construct(
     /**
      * BC layer for Symfony < 4.3
      */
-    public function handle(GetResponseEvent $event)
+    public function handle(RequestEvent $event)
```

Comment:
We don't need the handle method anymore since that was the BC layer for Symfony < 4.3 which we won't support on master after this gets merged,

Reply:
removed it.